i18n/vscode-language-pack-uk-UA/README.md

```diff
@@ -0,0 +1,108 @@
+# Языковой пакет для русского языка для VS Code
+
+Языковой пакет для русского языка содержит локализацию интерфейса VS Code.
+
+## Использование
+
+Вы можете переопределить язык пользовательского интерфейса по умолчанию, явно задав язык отображения VS Code с помощью команды «Настроить язык отображения».
+
+Нажмите <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> (<kbd>Command ⌘</kbd> + <kbd>Shift ⇧</kbd> + <kbd>P</kbd> на Mac), чтобы открыть палитру команд, а затем начните вводить «отображен…» (“display”), чтобы отфильтровать команду «Настроить язык отображения» (“Configure Display Language”). Нажмите клавишу <kbd>Ввод</kbd> (<kbd>Enter</kbd>), и откроется список установленных языков по языковым стандартам, в котором будет выбран текущий языковой стандарт. Чтобы изменить язык пользовательского интерфейса, выберите другой языковой стандарт.
+
+Дополнительные сведения: [Документация](https://go.microsoft.com/fwlink/?LinkId=761051).
+
+## Участие
+
+Чтобы отправить отзыв об улучшении перевода, создайте вопрос в репозитории [vscode-loc](https://github.com/microsoft/vscode-loc).
+Строки перевода находятся на платформе локализации Майкрософт. Изменения можно внести только на ней, а затем экспортировать в репозиторий vscode-loc. Поэтому запросы на вытягивание в репозитории vscode-loc приниматься не будут.
+
+## Лицензия
+
+На исходный код и строки распространяется лицензия [MIT](https://github.com/Microsoft/vscode-loc/blob/master/LICENSE.md).
+
+## Благодарность
+
+Языковой пакет для русского языка доступен вам благодаря усилиям сообщества.
+
+Выражаем особую признательность тем, кто принимал участие в создании пакета.
+
+**Участники:**
+
+* Mikhail Zabaluev
+* Ivan
+* Svitlana Galianova
+* Aleksey Nemiro
+* sberbanker
+* Nikita Gryzlov
+* Veronika Kolesnikova
+* Valeriy Schepak
+* Kirill Moskvichev
+* Alexander Filimonov
+* Anton Afonin
+* Serge Rodionov
+* Артем Мельниченко
+* Orrchy
+* Roman
+* Acupofspirt
+* Артём Давыдов
+* Andrei Makarov
+* Romanenko Alexey
+* Roman Sokolov
+* Andrei Pryymak
```

Comment:
Have these guys had anything to do with this language pack? No. Whoever has, it would be best to at least list their e-mails if not also GitHub usernames.